cranelift/frontend/src/ssa.rs

```diff
@@ -622,7 +618,7 @@ impl SSABuilder {
                     jt = func.create_jump_table(copied);
                 }
 
-                // Redo the match from `analyze_branch` but this time capture mutable references
+                // Redo the match from, but this time capture mutable references
```

Comment:
```suggestion
                // Redo the match from above, but this time capture mutable references
```

Reply:
Yeah, that does read a bit better :sweat_smile: